assignment/utils/formatDuration.js

```diff
@@ -0,0 +1,8 @@
+export const formatDuration = (milliseconds) => {
+  const minutes = Math.floor(milliseconds / 60000);
+  let seconds = (milliseconds % 60000) / 1000;
+  if (seconds.toString().length === 1) {
+    seconds = `0${seconds.toString()}`;
```

Comment:
You worked out a great solution here. You could also have used [padStart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart).